pkg/util/etcdutil/migrate.go

```diff
@@ -80,8 +82,8 @@ func waitEtcdTPRReady(httpClient *http.Client, interval, timeout time.Duration,
 func getBootEtcdPodIP(kubecli clientset.Interface) (string, error) {
 	var ip string
 	err := wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
-		podList, err := kubecli.Core().Pods(api.NamespaceSystem).List(api.ListOptions{
-			LabelSelector: labels.SelectorFromSet(labels.Set{"k8s-app": "boot-etcd"}),
+		podList, err := kubecli.CoreV1().Pods(api.NamespaceSystem).List(v1.ListOptions{
+			LabelSelector: labels.SelectorFromSet(labels.Set{"k8s-app": "boot-etcd"}).String(),
```

Comment:
If it just expects the selector string I believe we should be able to just use `"k8s-app=boot-etcd"`